README.md

```diff
@@ -91,16 +93,51 @@ $ yarn prepare
 $ yarn dev
 ```
 
-## Testing a production build
+## Running the moderation panel locally
+<details>
+<summary>Click to for instructions on running moderation panel locally</summary>
+ - You need to clone and setup the backend following its [README](https://github.com/ourjapanlife/findadoc-server).
+
+ Then for the backend run:
+
+```bash
+# install dependencies
+$ npm install
 
+# start the local db
+$ npm run dev:startlocaldb
+
+# For Development, start a dev server backend
+$ yarn dev
```

Comment:
The backend doesn't use `yarn` so they have to run `npm run dev` to start the server. Probably just a spelling mistake 😅

Reply:
Ahh it doesn't but in my machine it doesn't care which I use so I was definitely just thinking of what I type 😅